modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/translation/TranslationModal.js

```diff
@@ -88,12 +93,90 @@ const TranslationModal = ({
 					<ClayManagementToolbar.ItemList>
 						<ClayDropDown
 							active={active}
+							hasLeftSymbols
 							onActiveChange={setActive}
-							trigger={<ClayButtonWithIcon symbol="plus" />}
-						/>
+							trigger={
+								<ClayButtonWithIcon
+									disabled={deletedLocales.length < 1}
+									symbol="plus"
+								/>
+							}
+						>
+							<ClayDropDown.ItemList>
+								{deletedLocales.map(({index, locale}) => {
+									return (
+										<ClayDropDown.Item
+											key={index}
+											onClick={() => {
+												var deletedLocaleIndex = deletedLocales.indexOf(
```

Comment:
I think this `var` is not intended to be here. Also, What do you think in using a Set for deletedLocales? Since locales cannot be duplicated and it will save some LOCs to find and remove a given locale.

Reply:
@diegonvs you're looking at an outdated commit/diff, `var` has been replaced by `const` here. 
As for using `Set` sure, I'll do that

Reply:
Updated in [https://github.com/liferay-frontend/liferay-portal/commit/32d7ab2a9c6aea8787bffa20915918b37673b077](https://github.com/liferay-frontend/liferay-portal/commit/32d7ab2a9c6aea8787bffa20915918b37673b077)